client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -126,6 +126,7 @@ import {
   DpLabel,
   DpSelect,
   DpTextArea,
+  hasAnyPermissions,
```

Comment:
please put below `dpValidateMixin` to order alphabetically

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/4130/commits/c5c4f42730068abb4536abcc58b2a960130effb2